libavformat/dashdec.c

```diff
@@ -2316,7 +2316,7 @@ static int dash_read_packet(AVFormatContext *s, AVPacket *pkt)
     AVDictionary* metadata_dict = NULL;
     uint8_t* metadata_dict_packed = NULL;
```

Comment:
Corrupted stack if we use int instead of size_t!!! The method signature for av_packet_pack_dictionary changed